packages/adapters/relayer/src/gelato/gelato.ts

```diff
@@ -275,8 +275,14 @@ export const send = async (
 
   // Future intented way to call
   //const response = await gelatoSDKSend(request, gelatoApiKey, { gasLimit: GAS_LIMIT_FOR_RELAYER });
-
-  const response = await gelatoSDKSend({ chainId, data: encodedData, target: destinationAddress }, gelatoApiKey);
+  let response;
+  if (chainId == 59140) {
```

Comment:
Why not just add 59140 to GAS_LIMIT_FOR_RELAYER? Do we not want to use the default for it?

Reply:
Coz the is only used in V0, and we only need it in this LH path.